src/components/project/project.rules.ts

```diff
@@ -472,6 +472,11 @@ export class ProjectRules {
               type: TransitionType.Neutral,
               label: 'Will Not Change Plan',
             },
+            {
+              to: ProjectStep.ActiveChangedPlan,
+              type: TransitionType.Changeset,
```

Comment:
It would probably be better to use a separate property here. The transitions inside of changesets still could have "types" so we don't want to block or overload that functionality. Maybe `changeset: true` or `inChangeset: true`.